drivers/clock_control/clock_stm32_ll_common.c

```diff
@@ -376,11 +378,11 @@ static int stm32_clock_control_init(struct device *dev)
 	/* Enable HSE if not enabled */
 	if (LL_RCC_HSE_IsReady() != 1) {
 		/* Check if need to enable HSE bypass feature or not */
-#ifdef CONFIG_CLOCK_STM32_HSE_BYPASS
-		LL_RCC_HSE_EnableBypass();
-#else
-		LL_RCC_HSE_DisableBypass();
-#endif /* CONFIG_CLOCK_STM32_HSE_BYPASS */
+		if (IS_ENABLED(CONFIG_CLOCK_STM32_HSE_BYPASS) {
```

Comment:
")" Is missing. It will cause a compile error.

Reply:
Thanks!